packages/node_modules/@webex/webex-core/src/credentials-config.js

```diff
@@ -32,6 +32,18 @@ const CredentialsConfig = AmpState.extend({
       */
     authorizationString: ['string', false, process.env.WEBEX_AUTHORIZATION_STRING || process.env.AUTHORIZATION_STRING],
 
+    /**
+     * Authorization URL which prompts for user's password. Inferred from
+     * {@link config.credentials.authorizationString}.
```

Comment:
Could you add a note that this config will be automatically set if `authorizationString` is set?

Reply:
Implementing this.